Dockerfile.build

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG BASE=golang:1.12-alpine
+FROM ${BASE}
+
+ARG ALPINE_PKG_BASE="make git gcc libc-dev libsodium-dev zeromq-dev"
```

Comment:
Not sure if all of this is needed. I will pull out all but make and git and see if it still builds. These seem to be copied from another repository.

Reply:
The build succeeded without the extra packages.